### DISCUSSION TITLE
Multiple Grabbers on one Raspi

### DISCUSSION THREAD

**Discussion by FalkPe** [hyperion-project/hyperion.ng#1134] [General]
Hello
Is it possible to use 2 grabbers on one raspberry pi? I use hyperion.ng.
Thanks

**Comment by Paulchen-Panther** [comment]
Why do you want to start Hyperion twice?
Or why two grabbers?

**Reply by FalkPe** [reply]
Because I have the hardware and I want to use different devices without a HDMI Switch (VU and a Chromecast).

**Reply by Paulchen-Panther** [reply]
Ahhh. I haven't read anything from external grabbers in your start post.
Until now it hasn't been implemented in Hyperion. If you want you can help us. 😄
